tiledb/sm/query/query.cc

```diff
@@ -1028,6 +1028,34 @@ Status Query::set_subarray(const void* subarray) {
   return Status::Ok();
 }
 
+Status Query::set_subarray(const tiledb::sm::Subarray* subarray) {
+  auto query_status = status();
+  if (query_status != tiledb::sm::QueryStatus::UNINITIALIZED &&
+      query_status != tiledb::sm::QueryStatus::COMPLETED) {
+    // note:
+    // Can be in this initialized state when query has been de-serialized
+    // server-side and are trying to perform local submit...
```

Comment:
End with a single period, do not need an ellipsis here.

Reply:
Done